compiler/luci-interpreter/src/CMakeLists.txt

```diff
@@ -25,14 +25,22 @@ message(STATUS "LUCI INTERPTER INITALIZED")
 set(SOURCES
     "${LUCI_INTERPRETER_INCLUDE_DIR}/luci_interpreter/Interpreter.h"
     Interpreter.cpp "${LUCI_INTERPRETER_INCLUDE_DIR}/luci_interpreter/SimpleMemoryManager.h" SimpleMemoryManager.cpp
-        "${LUCI_INTERPRETER_INCLUDE_DIR}/luci_interpreter/TestMemoryManager.h" TestMemoryManager.cpp)
+        "${LUCI_INTERPRETER_INCLUDE_DIR}/luci_interpreter/TestMemoryManager.h" TestMemoryManager.cpp
+        "${LUCI_INTERPRETER_INCLUDE_DIR}/luci_interpreter/BuddyMemoryManager.h" BuddyMemoryManager.cpp)
 
 if (NOT LUCI_INTERPRETER_STATIC)
   add_library(${LUCI_INTERPRETER_BINARY} SHARED ${SOURCES})
 else ()
   add_library(${LUCI_INTERPRETER_BINARY} STATIC ${SOURCES})
 endif ()
 
+nnas_find_package(GTest REQUIRED)
```

Comment:
Could you move this to the bottom of CMakeLists.txt and add condition?
``` c++
if(NOT ENABLE_TEST)
  return()
endif(NOT ENABLE_TEST)
```

Reply:
Done